python/tk_framework_desktopserver/shotgun/api_v1.py

```diff
@@ -9,7 +9,7 @@
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
 
-class ShotgunAPI():
+class ShotgunAPI(object):
```

Comment:
It would be tough to do that and maintain backwards compatibility with an older version of Shotgun. It would require adapting old-style RPC command names, which don't match what we consider convention now (mixed case instead of underscores), to new-convention method names.

I would rather see us deprecate support for the v1 API at some point in the future and just ditch the v1 API altogether.